src/main/java/labs/introtoprogramming/lab5/object/Builder/POJO/VertexFace.java

```diff
@@ -0,0 +1,48 @@
+package labs.introtoprogramming.lab5.object.Builder.POJO;
+
+/**
+ * class that describes structure of vertex
+ */
+public class VertexFace {
+    private int index = -1;
+    private VertexGeometry vertexG = null;
+    private VertexTexture vertexT = null;
+    private VertexNormal vertexN = null;
```

Comment:
If you need vector with only 2 coordinates, use Vector3 with zero z value or write Vector2 class.